rdk/template/runtime/python3.6/rule_code.py

```diff
@@ -149,6 +149,12 @@ def build_evaluation_from_config_item(configuration_item, compliance_type, annot
 # Boilerplate Code #
 ####################
 
+# Helper function to build annotation
+def build_annotation(annotation_string):
+    if len(annotation_string) > 256:
+        return annotation_string[0:242] + "[...truncated]"

```

Comment:
- remove the `...` and a space in front to just have `" [truncated]"`
- integrate it with the functions `build_evaluation` and `build_evaluation_from_ci()`
- remove the 0 from `annotation_string[0:242]`. it is not required.

Reply:
Done